docs/install/common-service-integration.md

```diff
@@ -9,12 +9,21 @@
     - [1. Create OperatorSource](#1-create-operatorsource)
     - [2. Create a Namespace for your operator](#2-create-a-namespace-for-your-operator)
     - [3. Install Operator](#3-install-operator)
+- [Resolving services dependency](#resolving-services-dependency)
+  - [How to share secret or configmap between services](#how-to-share-secret-or-configmap-between-services)
+    - [Background](#background)
+    - [As an information provider](#as-an-information-provider)
+    - [As a information requester/adopter](#as-a-information-requesteradopter)
+    - [One more thing](#one-more-thing)
+  - [Add required CRD into CSV](#add-required-crd-into-csv)
+    - [Required CRDs](#required-crds)
 - [Common Service Onboarding](#common-service-onboarding)
   - [1. Clone the git repository of meta operator](#1-clone-the-git-repository-of-meta-operator)
   - [2. Edit default value of custom resource](#2-edit-default-value-of-custom-resource)
     - [Edit the MetaOperator Catalog custom resource](#edit-the-metaoperator-catalog-custom-resource)
```

Comment:
should update the ns to `ibm-common-services`.

Reply:
Notify the users to use `ibm-common-services` namespace here https://github.com/IBM/meta-operator/pull/99/files#diff-741e69926f77a1716aba252422e246bcR234